estate/models/estate_property.py

```diff
@@ -0,0 +1,47 @@
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Create a new table"
+
+    # FIELDS DECLARATION #
+    name = fields.Char(
+        string='Name', 
+        required=True
+    )
+    description = fields.Text(string='Description')
+    postcode = fields.Char(string='Post code')
+    date_availability = fields.Date(
+        string='Availability date',
+        default=fields.Date.add(fields.Date.today(), months=3),
+        copy=False 
+    )
+    expected_price = fields.Float(string='Expected price', required=True)
+    selling_price = fields.Float(
+        string='Selling price',
+        readonly=True,
+        copy=False
+    )
+    bedrooms = fields.Integer(
+        string='Bedrooms',
+        default=2
+    )
+    living_area = fields.Integer(string='Living area')
+    facades = fields.Integer(string='Facades')
+    garage = fields.Boolean(string='Garage')
+    garden = fields.Boolean(string='Garden')
+    garden_area = fields.Integer(string='Garden area')
+    garden_orientation = fields.Selection( 
+        string='Garden orientation',
+        selection= [('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')]
+    )
+    active = fields.Boolean(
+        string='Active',
+        default=True
+    )
+    state = fields.Selection(
+        string='State',
+        selection= [('new', 'New'), ('offer_received', 'Offer Received'), ('offer_accepted', 'Offer Accepted'), ('sold', 'Sold'), ('cancelled', 'Cancelled')],
+        default='new'
+    )
```

Comment:
```suggestion
    name = fields.Char(required=True)
    description = fields.Text()
    postcode = fields.Char()
    date_availability = fields.Date(
        string='Availability date',
        default=fields.Date.add(fields.Date.today(), months=3),
        copy=False 
    )
    expected_price = fields.Float(required=True)
    selling_price = fields.Float(
        readonly=True,
        copy=False
    )
    bedrooms = fields.Integer(default=2)
    living_area = fields.Integer()
    facades = fields.Integer()
    garage = fields.Boolean()
    garden = fields.Boolean()
    garden_area = fields.Integer()
    garden_orientation = fields.Selection( 
        selection= [('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')]
    )
    active = fields.Boolean(default=True)
    state = fields.Selection(
        selection= [('new', 'New'), ('offer_received', 'Offer Received'), ('offer_accepted', 'Offer Accepted'), ('sold', 'Sold'), ('cancelled', 'Cancelled')],
        default='new'
    )
```
Here the string are correctly generated from the variable names,no need for them :)